packages/appconfig/index.tst.ts

```diff
@@ -0,0 +1,164 @@
+import { AppConfigDataClient } from "@aws-sdk/client-appconfigdata";
+import middy from "@middy/core";
+import { getInternal } from "@middy/util";
+import type { Context as LambdaContext } from "aws-lambda";
+import { captureAWSv3Client } from "aws-xray-sdk";
+import { expect, test } from "tstyche";
+import appConfig, { appConfigReq } from ".";
```

Comment:
How can it be that extension less imports are still allowed? Ah.. The root `./tsconfig.json` makes this work. It defaults to CJS resolution. Is that useful for any other tooling?

I see two options:

- `npm run test:types:tstyche -- --tsconfig ignore`, simply use TSTyche defaults;
- extend the root TSConfig from [`@tsconfig/node20`](https://www.npmjs.com/package/@tsconfig/node20), since Node.js 20 is the lowest supported version

Reply:
The extension less imports should fail after badad62. (Many thanks to @willfarrell!)